core/src/test/java/org/infinispan/test/TestingUtil.java

```diff
@@ -354,15 +354,15 @@ public static void waitForNoRebalance(Cache... caches) {
             boolean chContainsAllMembers;
             boolean currentChIsBalanced;
             if (cacheTopology != null) {
-               rebalanceInProgress = cacheTopology.getPendingCH() != null;
+               rebalanceInProgress = cacheTopology.getPendingCH() != null || cacheTopology.getPhase() == CacheTopology.Phase.CONFLICT_RESOLUTION;
```

Comment:
I think we could use `cacheTopology.getPhase() != NO_REBALANCE`

Reply:
If it wasn't clear from the change, the change to `currentCH.locateOwnersForSegment(i).size() != actualNumOwners` was necessary as the CR+Rebalance topology update goes as follows:

1.CONFLICT_RESOLUTION
2.NO_REBALANCE
3.READ_OLD_WRITE_ALL

So some of the tests were failing because `waitForNoRebalance` was returning during stage 2.

Reply:
I actually missed that change, I was only focusing on the pending CH test, but looks ok to me.

I'm not sure why I used `<` there, but I don't see any reason to keep using it now.